### PR TITLE
Update Readme to have docs PDF build instructions

### PR DESCRIPTION
**Pull request by straithe**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ Information on HackRF and purchasing HackRF: https://greatscottgadgets.com/hackr
 
 Documentation for HackRF can be viewed on [Read the Docs](https://hackrf.readthedocs.io/en/latest/). The raw documenation files for HackRF are in the [docs folder](https://github.com/mossmann/hackrf/tree/master/docs) in this repository and can be built locally by installing [Sphinx Docs](https://www.sphinx-doc.org/en/master/usage/installation.html) and running `make html`. Documentation changes can be submitted through pull request and suggestions can be made as GitHub issues. 
 
+To create a PDF of the HackRF One documentation from the HackRf repository:
+* navigate to hackrf/docs on command line
+* run the command `make latex`
+* navigate to hackrf/docs/latex
```
Comment (mossmann) [inline Readme.md:23]: Based on my testing, this directory change is incorrect and unnecessary.

**Review by mossmann — CHANGES_REQUESTED** [review]
Please make one small change.
FYI, I had to `sudo apt install latexmk texlive-latex-extra` for this to work.

File: Readme.md (hunk 1 above)
Comment (straithe) [reply Readme.md]: Change made

**Comment by straithe** [comment]
> Please make one small change. FYI, I had to `sudo apt install latexmk texlive-latex-extra` for this to work.

Done

**Review by mossmann — APPROVED** [review]
awesome!

**Pull request merged**